packages/react-ui-screenshot-tests/gemini/ComboBox.js

```diff
@@ -6,10 +6,10 @@ gemini.suite('ComboBox', () => {
       suite
         .before(renderStory('ComboBoxView', 'input like text'))
         .setCaptureElements('#test-element')
-        .ignoreElements('[class^="Spinner-spinner"]')
+        .ignoreElements('[data-comp-name="Spinner"]')
         .capture('plain')
         .capture('focused first element', (action, find) => {
-          action.click(find('[class^="Input-input"]'));
+          action.click(find('[data-comp-name^="Input"]'));
```

Comment:
```suggestion
          action.click(find('[data-comp-name="Input"]'));
```

data-comp-name'ы же можно смело проверять на полное соответствие.

Reply:
В этом случае можно. Поправил.
Но есть места, где атрибут состоит из нескольких названий, например у `CurrencyInput` -  `data-comp-name="CurrencyInput Input"`. Получается завязка на структуру компонента. В этих случаях, кажется, лучше будет использовать более универсальные `^=` и `*=`, чтобы тесты не падали из-за структурных изменений.